### ISSUE TITLE
pylint incorrectly warns about threading.Lock.locked

### ISSUE THREAD

**Issue by vlovich** [PyCQA/pylint#2791]
### Steps to reproduce
1. test.py
```
import threading
l = threading.Lock()
l.locked()
```
2. pylint -E test.py

### Current behavior
************* Module test
test.py:3:0: E1101: Instance of 'lock' has no 'locked' member (no-member)

### Expected behavior
Shouldn't error because Lock is defined to have a locked() function.

### pylint --version output
pylint 2.3.1
astroid 2.2.3
Python 3.7.2 (default, Feb 13 2019, 15:31:10) 
[Clang 10.0.0 (clang-1000.11.45.5)]
